prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -47,21 +51,19 @@ def _list_buckets(self, provider):
                     ):
                         self.regions_with_buckets.append(bucket_region)
                         # Check if there are filter regions
-                        if provider.identity.audited_regions:
-                            # FIXME: what if the bucket comes from a CloudTrail bucket in another audited region
-                            if bucket_region in provider.identity.audited_regions:
-                                buckets.append(
-                                    Bucket(
-                                        name=bucket["Name"],
-                                        arn=arn,
-                                        region=bucket_region,
-                                    )
-                                )
+                        # FIXME: what if the bucket comes from a CloudTrail bucket in another audited region
+                        if (
```

Comment:
Please, revert this logic since we don't want to store the bucket if there are no `provider.identity.audited_regions` in any case.